Makefile

```diff
@@ -0,0 +1,26 @@
+MKFILE_PATH  := $(abspath $(lastword $(MAKEFILE_LIST)))
+ROOT_DIR := $(dir $(MKFILE_PATH))
+HACK_DIR := $(ROOT_DIR)/hack
+COMMON_SCRIPT := $(HACK_DIR)/common.sh
+LOCAL_ROOT := $(ROOT_DIR)/.local
+export LOCAL_ROOT
+
+.PHONY: cluster/new-name
+cluster/new-name:
+	@source $(COMMON_SCRIPT) && echo "$${USER}-prombenchmark-$$(date_w_format)"
```

Comment:
In this case `date_w_format` is defined in that script. That function is not a big deal but it is an easy way to produce a formatted date that is a valid cluster name, as it can be used as a DNS

The other targets also use things defined in that script:

- `cluster/list` uses `MON_BENCHMARKS_ROOT` to list the cluster configurations directory
- `cluster/kubeconfig` uses `cluster_config_dir`to get the root configuration directory for a given cluster

Reply:
Thank you so much for making it clear to me!